backend/coins/btc/account.go

```diff
@@ -716,7 +716,10 @@ func (account *Account) GetUnusedReceiveAddresses() []accounts.AddressList {
 	if !account.isInitialized() {
 		return nil
 	}
-	account.Synchronizer.WaitSynchronized()
+	// account.Synchronizer.WaitSynchronized()
+	if !account.Synced() {
+		return nil //TODO bznein: changed this function to return an error instead?
```

Comment:
yes why not error?

Reply:
I'm not sure as there is another line in this function that just return `nil` (just above) and one that panics (!). Do we want to change all of them to return an error instead?

Reply:
I see - I think the implicit assumption here is that this will not be called by the frontend until after it is synced anyway maybe. So we could keep the assumption and return `nil` (or panic) here too, or properly convert this to return an error instead and propagate it up (which is a bit annoying to do, could postpone this to another PR).